### ISSUE TITLE
It doesn't compile with Android Cordova 4.0.0

### ISSUE THREAD

**Issue by rromerogar** [phonegap/phonegap-mobile-accessibility#13]
When build a project with this plugin with Android Cordova 4.0.0 shows this errors:

```
:compileArmv7DebugJava/home/rromero/Devel/CVAPP/platforms/android/src/com/phonegap/plugin/mobileaccessibility/DonutMobileAccessibilityHelper.java:41: error: incompatible types: CordovaWebView cannot be converted to WebView
        mWebView = mobileAccessibility.webView;
                                      ^
/home/rromero/Devel/CVAPP/platforms/android/src/com/phonegap/plugin/mobileaccessibility/MobileAccessibility.java:135: error: cannot find symbol
                        webView.reload();
                               ^
  symbol:   method reload()
  location: variable webView of type CordovaWebView
/home/rromero/Devel/CVAPP/platforms/android/src/com/phonegap/plugin/mobileaccessibility/JellyBeanMobileAccessibilityHelper.java:37: error: cannot find symbol
        mParent = mobileAccessibility.webView.getParentForAccessibility();
                                             ^
  symbol:   method getParentForAccessibility()
  location: variable webView of type CordovaWebView
Note: Some input files use or override a deprecated API.
Note: Recompile with -Xlint:deprecation for details.
Note: Some input files use unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
3 errors
 FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileArmv7DebugJava'.
> Compilation failed; see the compiler error output for details.
```

**Comment by rromerogar** [comment]
This patch solve the problem:

```
diff --git src/android/com/phonegap/plugin/mobileaccessibility/DonutMobileAccessibilityHelper.java src/android/com/phonegap/plugin/mobileaccessibility/DonutMobileAccessibilityHelper.java
index f8a19c1..6381989 100644
--- src/android/com/phonegap/plugin/mobileaccessibility/DonutMobileAccessibilityHelper.java
+++ src/android/com/phonegap/plugin/mobileaccessibility/DonutMobileAccessibilityHelper.java
@@ -38,7 +38,7 @@ public class DonutMobileAccessibilityHelper extends
     @Override
     public void initialize(MobileAccessibility mobileAccessibility) {
         mMobileAccessibility = mobileAccessibility;
-        mWebView = mobileAccessibility.webView;
+        mWebView = (WebView) mobileAccessibility.webView;
         mAccessibilityManager = (AccessibilityManager) mMobileAccessibility.cordova.getActivity().getSystemService(Context.ACCESSIBILITY_SERVICE);
     }
 
diff --git src/android/com/phonegap/plugin/mobileaccessibility/JellyBeanMobileAccessibilityHelper.java src/android/com/phonegap/plugin/mobileaccessibility/JellyBeanMobileAccessibilityHelper.java
index 9aceb67..afcbee4 100644
--- src/android/com/phonegap/plugin/mobileaccessibility/JellyBeanMobileAccessibilityHelper.java
+++ src/android/com/phonegap/plugin/mobileaccessibility/JellyBeanMobileAccessibilityHelper.java
@@ -26,6 +26,7 @@ import com.phonegap.plugin.mobileaccessibility.MobileAccessibility;
 import android.annotation.TargetApi;
 import android.os.Build;
 import android.view.accessibility.AccessibilityEvent;
+import android.webkit.WebView;
 
 @TargetApi(Build.VERSION_CODES.JELLY_BEAN)
 public class JellyBeanMobileAccessibilityHelper extends
@@ -34,7 +35,7 @@ public class JellyBeanMobileAccessibilityHelper extends
     @Override
     public void initialize(MobileAccessibility mobileAccessibility) {
         super.initialize(mobileAccessibility);
-        mParent = mobileAccessibility.webView.getParentForAccessibility();
+        mParent = ((WebView) mobileAccessibility.webView).getParentForAccessibility();
     }
 
     @Override
diff --git src/android/com/phonegap/plugin/mobileaccessibility/MobileAccessibility.java src/android/com/phonegap/plugin/mobileaccessibility/MobileAccessibility.java
index 693f5bf..d732517 100644
--- src/android/com/phonegap/plugin/mobileaccessibility/MobileAccessibility.java
+++ src/android/com/phonegap/plugin/mobileaccessibility/MobileAccessibility.java
@@ -31,6 +31,7 @@ import org.json.JSONException;
 import org.json.JSONObject;
 
 import android.os.Build;
+import android.webkit.WebView;
 
 /**
  * This class provides information on the status of native accessibility services to JavaScript.
@@ -132,7 +133,7 @@ public class MobileAccessibility extends CordovaPlugin {
             stop();
             cordova.getActivity().runOnUiThread(new Runnable() {
                 public void run() {
-                        webView.reload();
+                        ((WebView) webView).reload();
                     }
                 });
         }
```

**Comment by majornista** [comment]
Fixed with pull request #11.
